<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotchFilterDlg</class>
 <widget class="QDialog" name="NotchFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notch Filter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="Kwave::ScaleWidget" name="scale_db" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Kwave::FrequencyResponseWidget" name="freq_response" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Kwave::ScaleWidget" name="scale_freq" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="KIntSpinBox" name="spinbox">
     <property name="toolTip">
      <string>center frequency</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;center frequency&lt;/b&gt;
Determines the center of the frequency range that
should be removed.</string>
     </property>
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="maxValue" stdset="0">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="slider">
     <property name="toolTip">
      <string>center frequency</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;center frequency&lt;/b&gt;
Determines the center of the frequency range that
should be removed.</string>
     </property>
     <property name="maximum">
      <number>44100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="slider_2">
     <property name="toolTip">
      <string>bandwidth</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;bandwidth&lt;/b&gt;
Determines the width of the frequency range that
should be removed.</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>44100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="KIntSpinBox" name="spinbox_2">
     <property name="toolTip">
      <string>bandwidth</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;bandwidth&lt;/b&gt;
Determines the width of the frequency range that
should be removed.</string>
     </property>
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="maxValue" stdset="0">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="frame3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KDialogButtonBox" name="buttonHelp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="btListen">
       <property name="toolTip">
        <string>start/stop pre-listening</string>
       </property>
       <property name="whatsThis">
        <string>Start or stop the pre-listening of the current settings. If pre-listening is enabled, you will nearly immediately hear any change in the selected parameters.</string>
       </property>
       <property name="text">
        <string>&amp;Listen</string>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>143</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>kdialogbuttonbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>Kwave::ScaleWidget</class>
   <extends>QWidget</extends>
   <header location="global">libgui/ScaleWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Kwave::FrequencyResponseWidget</class>
   <extends>QWidget</extends>
   <header>libgui/FrequencyResponseWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>slider</tabstop>
 </tabstops>
 <includes>
  <include location="global">klocale.h</include>
  <include location="local">libgui/ScaleWidget.h</include>
  <include location="local">libgui/FrequencyResponseWidget.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NotchFilterDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NotchFilterDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
